sky/serve/serve_utils.py

```diff
@@ -664,7 +677,8 @@ def _get_replica_status() -> serve_state.ReplicaStatus:
     handle = global_user_state.get_handle_from_cluster_name(
         replica_cluster_name)
     if handle is None:
-        return _FAILED_TO_FIND_REPLICA_MSG.format(replica_id=replica_id)
+        with ux_utils.print_exception_no_traceback():
+            return _FAILED_TO_FIND_REPLICA_MSG.format(replica_id=replica_id)
     assert isinstance(handle, backends.CloudVmRayResourceHandle), handle
```

Comment:
Do we really need this?

Reply:
Oops... Forgot to push the commit that removed this one